src/Components/Login/Login.tsx

```diff
@@ -96,6 +97,11 @@ const Login = () => {
          {activeAccount === true && <button className='login-btn' onClick={verifyLogin}>Login</button>}
          {activeAccount === false && <button className='login-btn' onClick={createUser}>Create Account</button>}
       </div>
+      <div className='create-acct'>
+        {activeAccount === true && <h3>Don't have an account? </h3>}
+        {activeAccount === true && <button className='login-btn' onClick={() => setAccount(false)}>Create New Account</button>}
+        {activeAccount === false && <button className='login-btn' onClick={() => setAccount(true)}>Return to Login</button>}
```

Comment:
No change necessary, but this could be rewritten as:

```TS
{activeAccount === true && <h2>Don't have an account?</h2>}
<button className='login-btn' onClick={() => setAccount(!activeAccount)}>{activeAccount ? "Create New Account" : "Return to Login"}</button>
```